### PR TITLE
add qtypings-browser-global and qtypings-nodejs-global to types

### PR DESCRIPTION
**Pull request by qiushaocloud**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

If removing a declaration:
- [x] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [x] Delete the package's directory.
- [x] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by qiushaocloud** [comment]
add qtypings-browser-global and qtypings-nodejs-global to types

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/qtypings-nodejs-global/index.d.ts
1. types/qtypings-nodejs-global/package.json
1. types/qtypings-browser-global/index.d.ts
1. types/qtypings-browser-global/package.json
1. types/qtypings-nodejs-global/typings/fn.d.ts

<details>
<summary>as well as these 5 other files...</summary>
<p>types/qtypings-nodejs-global/typings/object.d.ts, types/qtypings-nodejs-global/typings/nodejs.d.ts, types/qtypings-browser-global/typings/browser.d.ts, types/qtypings-browser-global/typings/object.d.ts, types/qtypings-browser-global/typings/fn.d.ts</p>
</details>


Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 64529961c15a56324d420995de53c6591f4cd4c7
</p>

File: types/qtypings-nodejs-global/typings/fn.d.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+/** 是否成功回调函数, 第2个参数开始任意回调参数*/
+interface QFnIsSuccessAnyArgs{
+    (isSuccess: boolean, ...args: any[]):void;
+}
+
+/** 错误则无result，有result则无错误 */
+interface QFnErrorOrResult{
+    // eslint-disable-next-line @definitelytyped/no-any-union
+    (err: Error | any | undefined, result?: any, ...args: any[]): void;
```
Comment (sheetalkamat) [inline types/qtypings-nodejs-global/typings/fn.d.ts:29]: This signature will have result with parameter `err` typed as `any` so please fix that and remove the eslint disable line.

File: types/qtypings-browser-global/typings/fn.d.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+/** 是否成功回调函数, 第2个参数开始任意回调参数*/
+interface QFnIsSuccessAnyArgs{
+    (isSuccess: boolean, ...args: any[]):void;
+}
+
+/** 错误则无result，有result则无错误 */
+interface QFnErrorOrResult{
+    // eslint-disable-next-line @definitelytyped/no-any-union
+    (err: Error | any | undefined, result?: any, ...args: any[]): void;
```
Comment (sheetalkamat) [inline types/qtypings-browser-global/typings/fn.d.ts:29]: Same, please fix the error and remove the eslint disable here.

**Review by sheetalkamat — CHANGES_REQUESTED** [review]
Also please run format per https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68153#issuecomment-1884426928

**Review by sheetalkamat — CHANGES_REQUESTED** [review]
Also both the packages are missing test files that validate the package.

File: types/qtypings-browser-global/typings/fn.d.ts (hunk 2 above)
Comment (qiushaocloud) [reply types/qtypings-browser-global/typings/fn.d.ts]: bug fixed

File: types/qtypings-nodejs-global/typings/fn.d.ts (hunk 1 above)
Comment (qiushaocloud) [reply types/qtypings-nodejs-global/typings/fn.d.ts]: bug fixed

File: types/qtypings-browser-global/qtypings-browser-global-tests.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+/// <reference path="./index.d.ts" />
+
```
Comment (sheetalkamat) [inline types/qtypings-browser-global/qtypings-browser-global-tests.ts:2]:
```suggestion
```

You dont need for global tests

File: types/qtypings-nodejs-global/qtypings-nodejs-global-tests.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+/// <reference path="./index.d.ts" />
+
```
Comment (sheetalkamat) [inline types/qtypings-nodejs-global/qtypings-nodejs-global-tests.ts:2]:
```suggestion
```

You dont need for global tests

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: types/qtypings-nodejs-global/tsconfig.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true
+    },
+    "files": [
+        "index.d.ts"
```
Comment (sheetalkamat) [inline types/qtypings-nodejs-global/tsconfig.json:16]: include test file as well.

File: types/qtypings-browser-global/tsconfig.json [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true
+    },
+    "files": [
+        "index.d.ts"
```
Comment (sheetalkamat) [inline types/qtypings-browser-global/tsconfig.json:16]: Include test file as well

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

**Comment by qiushaocloud** [comment]
I only define some global types in the package I publish, such as QJson, to facilitate use in projects. There is no specific implementation, and I don't need test files. However, when I add test files, I consistently receive the error message: "Error: 1:1 error Packages should contain value components, not just types @definitelytyped/no-type-only-packages." I hope to publish without including test files. If that's not possible, please advise on the necessary modifications.

**Comment by qiushaocloud** [comment]
I am releasing a package that solely defines some global types, such as QJson, for convenient use in projects. There is no specific implementation, and no test files are needed.

1. If possible, I would like to publish without including test files. Is this feasible, and if so, what steps should I take to exclude test files from the release?

2. After submitting, the Continuous Integration (CI) process fails consistently, displaying the error: "Error: 1:1 error Packages should contain value components, not just types @definitelytyped/no-type-only-packages." Even after reverting the code, the failure persists. Consequently, I added "@definitelytyped/no-type-only-packages" to my .eslintrc.json. If this addition is not allowed, please provide guidance on the necessary modifications.

File: types/qtypings-nodejs-global/tsconfig.json (hunk 5 above)
Comment (qiushaocloud) [reply types/qtypings-nodejs-global/tsconfig.json]: I am releasing a package that solely defines some global types, such as QJson, for convenient use in projects. There is no specific implementation, and no test files are needed.

1. If possible, I would like to publish without including test files. Is this feasible, and if so, what steps should I take to exclude test files from the release?

2. After submitting, the Continuous Integration (CI) process fails consistently, displaying the error: "Error: 1:1 error Packages should contain value components, not just types @definitelytyped/no-type-only-packages." Even after reverting the code, the failure persists. Consequently, I added "@definitelytyped/no-type-only-packages" to my .eslintrc.json. If this addition is not allowed, please provide guidance on the necessary modifications.

File: types/qtypings-browser-global/tsconfig.json (hunk 6 above)
Comment (qiushaocloud) [reply types/qtypings-browser-global/tsconfig.json]: I am releasing a package that solely defines some global types, such as QJson, for convenient use in projects. There is no specific implementation, and no test files are needed.

1. If possible, I would like to publish without including test files. Is this feasible, and if so, what steps should I take to exclude test files from the release?

2. After submitting, the Continuous Integration (CI) process fails consistently, displaying the error: "Error: 1:1 error Packages should contain value components, not just types @definitelytyped/no-type-only-packages." Even after reverting the code, the failure persists. Consequently, I added "@definitelytyped/no-type-only-packages" to my .eslintrc.json. If this addition is not allowed, please provide guidance on the necessary modifications.

**Comment by qiushaocloud** [comment]
I am releasing a package that solely defines some global types, such as QJson, for convenient use in projects. There is no specific implementation, and no test files are needed.

If possible, I would like to publish without including test files. Is this feasible, and if so, what steps should I take to exclude test files from the release?

After submitting, the Continuous Integration (CI) process fails consistently, displaying the error: "Error: 1:1 error Packages should contain value components, not just types @definitelytyped/no-type-only-packages." Even after reverting the code, the failure persists. Consequently, I added "@definitelytyped/no-type-only-packages" to my .eslintrc.json. If this addition is not allowed, please provide guidance on the necessary modifications.

**Pull request closed without merging**